ptf/README.md

```diff
@@ -89,15 +89,18 @@ To install TRex onto your server, please refer to the
 * `upf/ptf/tests/linerate/common.py` file updated with proper MAC address values
   for `TREX_SRC_MAC`, `UPF_CORE_MAC`, and `UPF_ACCESS_MAC`
 
-2. Generate BESS Python protobuf files for gRPC library and PTF Dockerfile image
+2. Move into the `ptf` directory
+
```

Comment:
```bash
cd upf/ptf
```
Can we add above snippet to make it clear.

Reply:
Done!